pkg/app/image_remove.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
+	"github.com/tensorchord/envd/pkg/docker"
+)
+
+var CommandRemoveImage = &cli.Command{
+	Name:    "remove",
+	Aliases: []string{"r"},
+	Usage:   "Remove an envd image",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    "image",
+			Usage:   "Specify the image name to be removed",
+			Aliases: []string{"i"},
+		},
+		&cli.StringFlag{
```

Comment:
Could we support --image image-name:tag?

Reply:
So the `images remove` command should only have one `--image` flag, and no need for `--tag` flag.

The possible usage like the following.
```
envd images remove --image name  # with default tag `dev`
envd images remove --image name:tag
```

Reply:
The default tag should be `latest`, I think.

Reply:
```
envd images remove --image name  # with default tag `latest`
envd images remove --image name:tag
```

Reply:
But the default tag in `envd build` is `dev`. 

https://github.com/tensorchord/envd/blob/f26fc4ce7e42ed0e567cb492c41551f6c2f2c3b4/pkg/app/build.go#L197-L200

Reply:
> Could we support --image image-name:tag?

Supported, PTAL @gaocegege